spec/requests/health_care_applications_request_spec.rb

```diff
@@ -408,6 +408,32 @@ def self.expect_async_submit
           end
         end
       end
+
+      context 'with hca_use_facilities_API enabled' do
+        before do
+          Flipper.enable(:hca_use_facilities_API)
+        end
+
+        let(:params) do
+          test_veteran['vaMedicalFacility'] = '000'
+          {
+            form: test_veteran.to_json
+          }
+        end
+
+        let(:body) do
```

Comment:
🤔 I get the use of `let` for `params` so the override can go in to the `subject` definition, but is there a reason to use it for `body` vs defining it locally inside the spec?